app/components/attachment_button.js

```diff
@@ -328,6 +330,20 @@ export default class AttachmentButton extends PureComponent {
         }, 100);
     };
 
+    removeProfileImage = () => {
+        this.props.removeProfileImage();
+        this.props.navigator.dismissModal({
+            animationType: 'none',
+        });
+    };
+
+    isCustomProfileImage = () => {
+        if (this.props.profileImageUri.includes('?')) {
```

Comment:
this one will not cause a crash as the one above as you are checking for the prop before calling this function so you are good here